components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/internal/action/PreUpdateProfileActionExecutor.java

```diff
@@ -61,12 +62,8 @@
 public class PreUpdateProfileActionExecutor {
 
     private static final Log LOG = LogFactory.getLog(PreUpdateProfileActionExecutor.class);
-    private static final List<String> EXCLUDED_IDENTITY_CLAIMS = Arrays.asList(
-            "http://wso2.org/claims/identity/adminForcedPasswordReset",
-            "http://wso2.org/claims/identity/askPassword",
-            "http://wso2.org/claims/identity/verifyEmail",
-            "http://wso2.org/claims/identity/verifyMobile"
-    );
+    private final ClaimMetadataManagementService claimMetadataManagementService = SCIMCommonComponentHolder
+            .getClaimManagementService();
```

Comment:
Don't initiate the claim management service in an instance variable.
These services are available at runtime so invoke the holder and use it directly